README.md

```diff
@@ -1,37 +1,80 @@
-    - [Adding a new configuration](/docs/siembol_ui/how-tos/how_to_add_new_config_in_siembol_ui.md)
-    - [Submitting configurations](/docs/siembol_ui/how-tos/how_to_submit_config_in_siembol_ui.md)
-    - [Importing a sigma rule](/docs/siembol_ui/how-tos/how_to_import_sigma_rules.md)
-    - [Releasing configurations](/docs/siembol_ui/how-tos/how_to_release_configurations_in_siembol_ui.md)
-    - [Testing configurations](/docs/siembol_ui/how-tos/how_to_test_config_in_siembol_ui.md)
-    - [Testing release](/docs/siembol_ui/how-tos/how_to_test_release_in_siembol_ui.md)  
-    - [Adding links to the homepage](/docs/siembol_ui/how-tos/how_to_add_links_to_siembol_ui_home_page.md)
-    - [Setting up OAUTH2 OIDC](/docs/siembol_ui/how-tos/how_to_setup_oauth2_oidc_in_siembol_ui.md)
-    - [Modifying the layout](/docs/siembol_ui/how-tos/how_to_modify_ui_layout.md)
-    - [Managing applications](/docs/siembol_ui/how-tos/how_to_manage_applications.md)
-    - [Use ui-bootstrap file](/docs/siembol_ui/how-tos/how_to_use_ui_bootstrap_file.md)
-    - [Filter configs and save searches](/docs/siembol_ui/how-tos/how_to_filter_configs_and_save_searches.md)
-- Siembol services    
-    - [Setting up a service in the config editor rest](/docs/services/how-tos/how_to_set_up_service_in_config_editor_rest.md)
-    - [Alerting service](/docs/services/siembol_alerting_services.md)
-    - [Parsing service](/docs/services/siembol_parsing_services.md)
-        - [How to setup NetFlow v9 parsing](/docs/services/how-tos/how_to_setup_netflow_v9_parsing.md)
-    - [Enrichment service](/docs/services/siembol_enrichment_service.md)
-        - [Setting up an enrichment table](/docs/services/how-tos/how_to_set_up_enrichment_table.md)
-    - [Response service](/docs/services/siembol_response_service.md)
-        - [Writing a response plugin](/docs/services/how-tos/how_to_write_response_plugin.md)
-- [Siembol deployment](/docs/deployment/deployment.md)
-    - [Setting up ZooKeeper nodes](/docs/deployment/how-tos/how_to_set_up_zookeeper_nodes.md)
-    - [Setting up a GitHub webhook](/docs/deployment/how-tos/how_to_setup_github_webhook.md)
-    - [Tuning the performance of Storm topologies](/docs/deployment/how-tos/how_to_tune_performance_of_storm_topologies.md)
-    - [Setting up Kerberos for external dependencies](/docs/deployment/how-tos/how_to_set_up_kerberos_for_external_dependencies.md)
-    - [Customize Helm chart](/docs/deployment/how-tos/how_to_customize_helm_charts.md)
+Introduction
+------------
+
+Siembol is an open-source, real-time security information and event management tool developed in-house at G-Research.
+ 
+Siembol is designed to support the following:
+
+* to centralize both security data collecting and the monitoring of logs from different sources.
+* as a tool for detecting attacks or leaks by teams responsible for the system platform.
+
+For a full description, visit: [Introduction](/docs/introduction/introduction.md).
```

Comment:
it is an introduction and not full description?

Reply:
just changed the wording!